tests/affiliatedcertification/tests/affillated_certification_helm_chart.go

```diff
@@ -0,0 +1,77 @@
+package tests
+
+import (
+	"os/exec"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	tsparams "github.com/test-network-function/cnfcert-tests-verification/tests/affiliatedcertification/parameters"
+	"github.com/test-network-function/cnfcert-tests-verification/tests/globalhelper"
+	"github.com/test-network-function/cnfcert-tests-verification/tests/globalparameters"
+	"github.com/test-network-function/cnfcert-tests-verification/tests/utils/execute"
+)
+
+var _ = Describe("Affiliated-certification helm chart certification,", func() {
+	execute.BeforeAll(func() {
+		By("Define tnf config file")
+		err := globalhelper.DefineTnfConfig(
+			[]string{tsparams.TestHelmChartCertified},
+			[]string{tsparams.TestPodLabel},
+			[]string{},
+			[]string{})
+		Expect(err).ToNot(HaveOccurred(), "error defining tnf config file")
+
```

Comment:
the system we run the tests on doesn't have helm installed by default so we probably want to install it in BeforeAll similar to how we install istio for the platform-alteration-service-mesh tests https://github.com/test-network-function/cnfcert-tests-verification/blob/d3ef59b1ed0240901a4b569d490d63b7b9ae14aa/tests/platformalteration/tests/platform_alteration_service_mesh.go#L33